workflows-scripts/item-search-filter/README.md

```diff
@@ -0,0 +1,64 @@
+## Workflow steps:
```

Comment:
Can the main modules, those that are involved in these workflows, be listed with their version numbers, and explain which FOLIO release it should run against?

Can we give these workflows a version number (as we discussed - with a "tag version")?

Reply:
> Can the main modules, those that are involved in these workflows, be listed with their version numbers, and explain which FOLIO release it should run against?
Yes

> Can we give these workflows a version number (as we discussed - with a "tag version")?
I think this PR needs to merge to master first. Then, we can create a tag.